src/control/lib/spdk/spdk.go

```diff
@@ -64,25 +66,86 @@ func Rc2err(label string, rc C.int) error {
 	return nil
 }
 
+type EnvOptions struct {
+	MemSize      int // memory size in MB for DPDK
+	ShmID        int
+	PciWhitelist []string
+	PciBlacklist []string
+}
+
+func convertPciList(in []string, out **C.struct_spdk_pci_addr) error {
```

Comment:
If you end up using this function, it will help if it returns the number of pci devices, so that we can set opts.num_pci_addr after the call.